tests/integration/test_macho_parser.py

```diff
@@ -53,4 +53,6 @@ def test_hackernews_parsing(self, sample_app_path: Path) -> None:
 
         imported_symbols = macho_parser.get_imported_symbols()
         assert len(imported_symbols) == 2019
-        assert imported_symbols[0] == "_$s10Foundation4DateV6CommonE14timeAgoDisplaySSyF"
+        assert (
+            imported_symbols[0] == "_$s10Foundation4DateV6CommonE14timeAgoDisplaySSyF"  # noqa: E501
```

Comment:
what's the # noqa: E501 for?

Reply:
its the line too long lint error, weird that it didn't come up for you though. looks like i have it erroneously enabled on local. just bumped, will disable on local now